src/js/components/AppListItemComponent.jsx

```diff
@@ -215,6 +243,54 @@ var AppListItemComponent = React.createClass({
     );
   },
 
+  getDropdownMenu: function () {
+    if (!this.state.isActionsDropdownActivated) {
+      return null;
+    }
+
+    let props = this.props;
```

Comment:
props.model is used twice in the next lines it would be great to have a a model variable aswell.